### PR TITLE
Jb/ios remote speech

### PR DESCRIPTION
**Pull request by justbill2020**
####  Description

adding input box for remote when IOS device is used.
awesome credit goes to @jfccoding for getting us over the hump here... I took his code and tweaked it a bit to fit the overall ways we do things... 

####  Fixes Related issues
no issue currently
doesn't address #544 

####  Checklist
- [x] I have read and understand the CONTRIBUTIONS.md file
- [x] I have searched for and linked related issues
- [x] I have added config.schema.json file if config option are required.
- [x] I am NOT targeting master branch

### CODE REVIEW

File: remote/index.html [hunk 1]
```diff
@@ -29,25 +29,39 @@
 </nav>
 
 <!-- Begin page content -->
+
 <div class="container">
+<p id="demo">
```
Comment (evancohen) [inline remote/index.html]: Why are you wrapping everything in a `<p>`?

File: remote/index.html [hunk 2]
```diff
@@ -29,25 +29,39 @@
...
     <div class="row">
-        <div class="col-md-6 col-md-offset-3"><button id="speak" type="button" class="btn btn-primary btn-lg btn-block glyphicons">Speak!</button></div>
+        <div class="col-md-6 col-md-offset-3">
+            <div class="form-group hidden" id="command">
```
Comment (evancohen) [inline remote/index.html]: Why not just make the text box visible all of the time and put a button next to it that has a microphone?

File: remote/index.html [hunk 3]
```diff
@@ -66,7 +80,6 @@
             </div>
         </form>
     </div>
-</div>
```
Comment (evancohen) [inline remote/index.html]: You sure about that?

File: remote/index.html [hunk 4]
```diff
@@ -41,13 +49,15 @@
...
     </div>
+
     <div class="row">
         <div class="col-md-12">
             <div class="btn-group btn-group-justified" role="group" aria-label="Justified button group">
                 <a href="#" id="kiosk" class="btn btn-default" role="button">Full Screen</a>
                 <a href="#" id="reload" class="btn btn-default" role="button">Reload</a>
                 <a href="#" id="sleep" class="btn btn-default" role="button">Sleep</a>
                 <a href="#" id="wakeUp" class="btn btn-default" role="button">Wake Up</a>
+		        <a href="#" id=“commands” class="btn btn-default" role="button">Show help</a>
```
Comment (evancohen) [inline remote/index.html]: ocd

**Comment by evancohen** [comment]
Why merge it preemptively? If anyone desperately needs to be able to type into a box they can just check out this branch :P

File: remote/index.html (hunk 4 above)
Comment (justbill2020) [inline remote/index.html]: i don't show that white space... also i didn't add the commands button...

**Comment by justbill2020** [comment]
these tests are going to force me not to be so lazy @evancohen

**Pull request merged**